### DISCUSSION TITLE
How to scan markdown files for references?

### DISCUSSION THREAD

**Discussion by boyoungzheng** [retorquere/zotero-better-bibtex#2671] [Q&A] [answered]
I used pandoc to create the .md file from a .tex file. But it is gray when I want to select it.

<img width="495" alt="image" src="https://github.com/retorquere/zotero-better-bibtex/assets/21117824/326537b4-5887-4926-91cd-3364224f7b13">
 
BTW, the .aux file also gives me "missing entries", like zycyc's comment [here](https://forums.zotero.org/discussion/77365/create-bib-file-only-from-papers-cited-in-a-tex-file).

I even manually created a .aux file with
```
\citation{CITEKEY1}
\citation{CITEKEY2}
```
as suggested by the [official guide](https://retorque.re/zotero-better-bibtex/citing/aux-scanner/), I still got nothing after being prompted to give the tag name. 
The "right click of a collection" approach works well with the manually crated .aux file though.

**Comment by retorquere** [comment]
> But it is gray when I want to select it.

That means `pandoc` is not in your `PATH`. See the rest of the guide you linked to.

> I still got nothing after being prompted to give the tag name.

If you used that literally then you didn't get any results because you don't have any item in your library with the citekeys `CITEKEY1` or `CITEKEY2` I reckon.

**Reply by boyoungzheng** [reply]
The gray .md problem is solved. Thanks!

> If you used that literally then you didn't get any results because you don't have any item in your library with the citekeys CITEKEY1 or CITEKEY2 I reckon.

I am super sure I have all the items with the citekeys in my library. The proof is that "right click of a collection" approach worked well and created a subcollection with all the items. Although the right-click approach worked fine, I still want the tag approach.

**Reply by retorquere** [reply, marked as answer]
Then we're in bug tracking territory. For that you'll have to open an issue.
